pkg/k8s/apis/cilium.io/v2/clrp_types.go

```diff
@@ -217,8 +217,13 @@ func (pInfo *PortInfo) SanitizePortInfo(checkNamedPort bool) (uint16, string, lb
 	}
 	// Sanitize name
 	if checkNamedPort {
+		if pInfo.Name == "" {
+			return pInt, pName, protocol, fmt.Errorf("port %s in the local "+
+				"redirect policy spec must be named as there are multiple ports", pInfo.Port)
+
+		}
 		if !iana.IsSvcName(pInfo.Name) {
-			return pInt, pName, protocol, fmt.Errorf("valid port name is not present")
+			return pInt, pName, protocol, fmt.Errorf("port name %s isn't valid", pInfo.Name)
```

Comment:
Maybe: `port name %s isn't a valid service name (see: RFC 6335 Section 5.1. Service Name Syntax)` to make it explicit wrt what we expect?

Reply:
Good idea - I decided to keep it consistent with the Kubernetes terminology (IANA_SVC_NAME).

Reply:
(Should this additionally get a LRP doc update explicitly mentioning IANA_SVC_NAME?)

Reply:
Sure. @raymonddejong Do you mind taking care of this one?